src/Operation/CallOperation.php

```diff
@@ -3,6 +3,7 @@
 namespace Tsuka\DB\Operation;
 
 use Katana\Sdk\Action;
+use Tsuka\DB\Binder\BinderInterface;
 use Tsuka\DB\Binder\IdListBinder;
```

Comment:
Nice catch, I forgot to change the type hint to the interface, but then this namespace is no longer needed.